### PR TITLE
Create app to track downtimes of services

### PR DESCRIPTION
**Pull request by ibrechin**
This allows one to add a scheduled downtime for a service. The status
of all services is returned via the healthcheck, and if the end time
of the downtime is known then this is also included in the healthcheck
response.

### CODE REVIEW

**Comment by ibrechin** [comment]
retest this please

File: mtp_api/apps/service/healthchecks.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+from service.models import Downtime, SERVICES
+
+
+def get_service_downtime_healthcheck(service):
+    def check_downtime():
+        status = {'name': service}
+        active_downtime = Downtime.objects.active_downtime(service)
+        if active_downtime is not None:
+            status['status'] = False
+            status['downtime_end'] = active_downtime.end.isoformat()
```
Comment (ushkarev) [inline mtp_api/apps/service/healthchecks.py]: can't `active_downtime.end` be None here?

File: mtp_api/apps/service/migrations/0001_initial.py [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Downtime',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('service', models.CharField(choices=[('gov_uk_pay', 'GOV.UK Pay')], max_length=50)),
+                ('start', models.DateTimeField(blank=True, null=True)),
```
Comment (ushkarev) [inline mtp_api/apps/service/migrations/0001_initial.py]: is this migration outdated? `blank=True, null=True` looks like it should be different…

**Comment by ibrechin** [comment]
retest this please

**Comment by ibrechin** [comment]
@ushkarev thanks, fixed

**Pull request merged**